server/index.js

```diff
@@ -1,22 +1,25 @@
 const express = require('express');
 const app = express();
 const cors = require('cors');
+const path = require('path')
 const pool = require('./db');
 const get_points_exchanged = require('./rating_algo');
 
 // middleware
 app.use(cors());
 app.use(express.json());
+app.use(express.static(path.resolve(__dirname, '../build')));
 
 // routes
 
+// TODO why can't we just return an array rather than access it through .list field
 // returns the id, name and rating of each player
 // returns { "list" : [ {"pid" : player_ID, "pname" : player_name, "pr" : rating, "active" : active}, ... ] }
 app.get('/api/players', (req, res) => {
     pool.query("select player_id as pid, player_name as pname, rating as pr, active from ratings order by rating desc, player_id asc", (err, results) => {
         // console.log("results:\n"+JSON.stringify(results.rows)+"\n\n\n");
         if (err) { /*console.log("hey there was an error: \n" + err);*/ res.status(500).send("something went wrong"); return;}
-        let result = {"list": results.rows}
+        let result = results.rows;
```

Comment:
Any reason why we can't just return `results.rows`?

Reply:
i read somewhere online that it's bad practice to directly return an array and that the highest level of what i'm returning should be able to be parsed as a dictionary/hashmap. and if i'm just returning a list then i should have it as a field.

also it makes it easier for us to added other stuff later on if we need to. for example returning an array of objects immediately fails if i also need to return the number of objects that is returned. but in the way that i was doing it, it'd be as simple as just adding another field to the object, and the rest of ur code would not need to be changed in any way to account for this change.